EIPS/eip-1.md

```diff
@@ -185,6 +185,10 @@ The `created` header records the date that the EIP was assigned a number. Both h
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on.
 
+## Linking to External Resources
+
+Links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
```

Comment:
I'd suggest changing "external resources" to "non-normative external resources."